config/configstore/service.go

```diff
@@ -89,6 +95,14 @@ func (s service) GenerateTenant() (config.TenantConfiguration, error) {
 		return nil, err
 	}
 
+	// minor hack to set same p2p keys as node to tenant: Set the new tenant ID to copy of main tenant and create p2p keys
```

Comment:
I do not understand this, why not using the same `tc` ?

Reply:
because I'm using main tenant p2p keys with new tenants ID, so its not a consistent config. Should not be saved to db.